assets/app/scripts/controllers/create.js

```diff
@@ -30,15 +32,25 @@ angular.module('openshiftConsole')
 
     var templatesByTag = function() {
       $scope.templatesByTag = {};
+      // not doing this as a map since we are dealing with things across namespaces that could have collisions on name
+      $scope.otherTemplates = [];
       var fn = function(template) {
         if (template.metadata.annotations && template.metadata.annotations.tags) {
           var tags = template.metadata.annotations.tags.split(",");
+          var instantApp = false;
           angular.forEach(tags, function(tag){
             tag = $.trim(tag);
-            // not doing this as a map since we are dealing with things across namespaces that could have collisions on name
+            // use an array to avoid name collisions here as well
             $scope.templatesByTag[tag] = $scope.templatesByTag[tag] || [];
             $scope.templatesByTag[tag].push(template);
```

Comment:
if the tag is repeated, the template gets added multiple times (map by uid would fix that)